dev-packages/node-integration-tests/package.json

```diff
@@ -15,6 +22,7 @@
     "type-check": "tsc",
     "pretest": "run-s --silent prisma:init prisma:init:new",
     "test": "ts-node ./utils/run-tests.ts",
+    "jest": "jest --config ./jest.config.js",
```

Comment:
I added this simply because it allows you to call `yarn jest -t "name-of-test"` and get debug output. 

Hopefully we can migrate to just using the jest runner than our custom `run-tests.ts`